packages/app/src/hooks/useLocalExecutor.ts

```diff
@@ -101,7 +101,7 @@ export function useLocalExecutor() {
           },
           data: projectData,
         };
-
+        
```

Comment:
Hm we must have different editor settings, yours likes to preserve whitespace between lines, mine likes to trim